build.gradle

```
@@ -19,6 +19,10 @@ repositories {
 }
 
 dependencies {
+    //spring-boot-devtools
+    implementation group: 'org.springframework.boot', name: 'spring-boot-devtools', version: '2.3.12.RELEASE'
+    //mybatis-spring-boot-starter
+    implementation group: 'org.mybatis.spring.boot', name: 'mybatis-spring-boot-starter', version: '2.1.3'
```

```suggestion
    implementation 'org.springframework.boot:spring-boot-devtools'
    implementation 'org.mybatis.spring.boot:mybatis-spring-boot-starter:2.1.3'
```

이렇게 바꿔보는건 어땠을까 궁금합니다.

---

사이트에서 의존성을 복사해오다보니 group 및 속성들이 추가되어 있습니다.
각 속성들이 어떠한 형태로 사용되는지는 아직 모르며 한번 검색해보겠습니다.

확인